R/drive_file.R

```diff
@@ -7,68 +7,17 @@
 #' @export
 #'
 drive_file <- function(id = NULL, ...) {
-  request <- build_drive_file(id = id, ...)
+  request <- build_request(
+    endpoint = "drive.files.get",
+    token = drive_token(),
+    params = list(...,
```

Comment:
It *sort* of ends up working out (but I can add more structure around this) - if a user passes a `fileId` and the endpoint is not one that takes a fileId, they will receive a message that the fileId parameter wasn't use. If the user supplies a fileId and a fileId *is* expected, the fileId I've told it to use takes precedence, so their fileId essentially does nothing (but I don't tell them that, so maybe I should) (I've added a test for this in my next push)